jetbrains-gateway/main.tf

```diff
@@ -120,6 +140,11 @@ variable "jetbrains_ides" {
   }
 }
 
+data "http" "jetbrains_ide_versions" {
+  for_each = var.latest ? toset(var.jetbrains_ides) : toset([])
```

Comment:
`var.latest` condition ensures we do not call the HTTP provider if latest is not requested.